### ISSUE TITLE
[trac import 8/18/14 by dominic] Record Merger Tab order could be improved

### ISSUE THREAD

**Issue by devinbalkind** [sahana/eden#1243]
In the record merger UI, it would be better if Tab took one between fields of the same record rather than across to the <> control & then the adjacent record

**Comment by huynhv** [comment]
What's a "record merger"?

**Comment by flavour** [comment]
http://eden.sahanafoundation.org/wiki/UserGuidelines/Admin/RecordMerging

**Comment by huynhv** [comment]
I've been looking at merge.html but still cannot find where the tab order is defined. 
It could be in one of the foundation css sheets, but can someone point me in the right direction to locate this?

Thank you.

**Comment by flavour** [comment]
I think that currently it is undefined, so just defaults to the order the elements appear in the HTML code.
To define the order, tabindex attributes shoul be aded to the inputs.
http://webcheatsheet.com/html/controll_tab_order.php

**Comment by nursix** [comment]
Fixed by https://github.com/sahana/eden/commit/9e0e565b4223be79cac2d9f425a193cfd92e9bf8
